Apsitvarkom.Models/DTO/LocationDTO.cs

```diff
@@ -3,11 +3,13 @@
 /// <summary>
 /// Data Transfer Object for <see cref="Location" />
 /// </summary>
-public class LocationDTO
+public struct LocationDTO
 {
+    public LocationDTO(double? longitude, double? latitude) => (Longitude, Latitude) = (longitude, latitude);
```

Comment:
Why is this constructor needed? I think we've decided to create these types of classes like this:
```csharp
new LocationDTO(){
  longitude: 100,
  latitude: 100
}
```